File: src/test/java/org/jdc/template/model/database/main/individual/IndividualManagerTestModule.java
package org.jdc.template.model.database.main.individual;

import org.dbtools.android.domain.config.DatabaseConfig;
import org.jdc.template.model.database.TestMainDatabaseConfig;

import javax.inject.Singleton;

import dagger.Module;
import dagger.Provides;

@Module
public class IndividualManagerTestModule {
    public IndividualManagerTestModule() {
    }

    @Provides
    @Singleton
    DatabaseConfig provideDatabaseConfig() {
        return TestMainDatabaseConfig.getInstance();
    }
}

File: src/test/kotlin/org/jdc/template/model/database/main/individual/IndividualManagerTestModule.kt
package org.jdc.template.model.database.main.individual

import dagger.Module
import dagger.Provides
import org.dbtools.android.domain.config.DatabaseConfig
import org.jdc.template.model.database.TestMainDatabaseConfig
import javax.inject.Singleton

@Module
class IndividualManagerTestModule {
    @Provides
    @Singleton
    internal fun provideDatabaseConfig(): DatabaseConfig {
        return TestMainDatabaseConfig.instance
    }
}
